src/main/resources/data/shardthief/games/cyber_block.json

```diff
@@ -0,0 +1,12 @@
+{
+	"type": "shardthief:shard_thief",
+	"name": "Shard Thief: Cyberblock",
+	"map": {
+		"structure_id": "shardthief:cyberblock",
+		"spawn_y_offset": 4
+	},
+	"players": {
+		"min": 1,
+		"max": 21
```

Comment:
Why 21 in particular?

Reply:
I did a test with my discord, it felt balanced at 18-20

Reply:
But since it's small, it could be a small player count: 15-16